src/main/java/io/github/townyadvanced/townyresources/commands/TownyResourcesAdminCommand.java

```diff
@@ -89,7 +89,7 @@ private void parseReRollCommand(CommandSender sender) {
 		.sendTo(sender);
 	}
 
-	private void bypassExtractionLimitCommand(CommandSender sender, String[] args) {
+	private void bypassExtractionLimitCommand(CommandSender sender) {
 		if(!sender.hasPermission(TownyResourcesPermissionNodes.TOWNY_RESOURCES_BYPASS.getNode("townyresources.bypass")))
```

Comment:
not entirely sure how the whole permission node system works, couldn't figure out what was going on.